slack/install.sh

```diff
@@ -0,0 +1,4 @@
+
+#!/bin/sh
```

Comment:
Let's talk about why we don't use this in every file and what advantage or disadvantage that gives us